cmd/relayer/main.go

```diff
@@ -45,11 +46,11 @@ const (
 	flagDeploy           = "deploy"
 	flagLog              = "log-level"
 	// TODO: do we need this, or can we assume all swap relayers will need to be on the p2p network?
-	flagWithNetwork       = "with-network"
-	flagLibp2pKey         = "libp2p-key"
-	flagLibp2pPort        = "libp2p-port"
-	flagBootnodes         = "bootnodes"
-	flagRelayerCommission = "relayer-commission"
+	flagWithNetwork = "with-network"
+	flagLibp2pKey   = "libp2p-key"
+	flagLibp2pPort  = "libp2p-port"
+	flagBootnodes   = "bootnodes"
+	flagRelayerFee  = "relayer-fee"
```

Comment:
What if we just called this `--min-fee`? It is the `relayer` executable, so the word `relayer` probably isn't adding any context for the end user.